### PR TITLE
Rate limiting syslog warnings about incorrect format

### PR DESCRIPTION
**Pull request by kirillt**
Added rate-limited warning to inform user about invalid or unsupported format.

Ref #741

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -118,7 +120,8 @@ fn main() {
     let subscriber = tracing_fmt::FmtSubscriber::builder()
         .with_ansi(color)
         .with_filter(tracing_fmt::filter::EnvFilter::from(levels.as_str()))
-        .finish();
+        .finish()
+        .with(Limit::default());
```
Comment (LucioFranco) [inline src/main.rs]: I am currently in the process of updating all of this right now, so I would hold off on any specific tracing PRs for the moment.

**Comment by LucioFranco** [comment]
If you want to open a separate PR for the docs that is fine.

File: src/main.rs (hunk 1 above)
Comment (kirillt) [reply src/main.rs]: Oh, ok, I can adapt this PR later

**Comment by LucioFranco** [comment]
@kirillt actually can you comment on the rate limiting issue with what this PR wants to change, and I can include that change in my Pr. I think that would be easier. Then we can close this one.

**Comment by kirillt** [comment]
@LucioFranco basically, I only need to rate-limit warnings about invalid Syslog format and that's all.

**Comment by LucioFranco** [comment]
@kirillt ok, then we should add that to #628

**Comment by kirillt** [comment]
@LucioFranco sounds good

**Comment by binarylogic** [comment]
I've added this as a requirement. Closing as a result. Thanks @kirillt ,

**Pull request closed without merging**